mne/viz/topomap.py

```diff
@@ -1134,19 +1135,34 @@ def plot_evoked_topomap(evoked, times=None, ch_type=None, layout=None,
 
     if isinstance(axes, plt.Axes):
         axes = [axes]
-    if times is None:
+
+    if times == "peaks":
+        from scipy.signal import argrelmax
+        gfp = evoked.data.std(axis=0)
+        order = int(evoked.info["sfreq"] / 5.)
+        peaks = argrelmax(gfp, order=order)
+        if len(peaks) > 10:
+            peaks = peaks[np.sort((-peaks).argsort()[:10])]
```

Comment:
Heh, I just enjoy littly problems like this :)

So if you want to make sure they're spaced out some reasonable amount (I think that's what you're going for with the `argrelmax`), then I think it you could do:
```
peaks = argrelmax(gfp, order=order)
peaks = np.sort(peaks[np.argsort(gfp[peaks])[::-1][:3])[:10]
```
So actually I think the only problem with your original solution was that you argsort'ed `peaks` when you meant to argsort `gfp[peaks]`.

And now that I think about it, you probably also want to set the `order` based on some multiple of the sample rate divided by the lowpass frequency, not the sample rate itself. This actually makes sense to me as a solution as being better than the simpler one @agramfort suggests, but it might also be overkill, especially given the challenges associated with getting it right :)